app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardGeneralStep/helpers.js

```diff
@@ -0,0 +1,30 @@
+export const asyncValidate = (values, dispatch, props) =>
+  new Promise((resolve, reject) => {
+    const { name: newPlanName } = values;
+    const { transformationPlans, archivedTransformationPlans } = props;
+    const existingTransformationPlanNames = transformationPlans.reduce(
+      (names, plan) => [...names, plan.name.trim()],
+      []
+    );
+    const existingArchivedPlanNames = archivedTransformationPlans.reduce(
+      (names, plan) => [...names, plan.name.trim()],
+      []
+    );
+
+    const allPlanNames = [...existingTransformationPlanNames, ...existingArchivedPlanNames];
+    const duplicateName = allPlanNames.find(existingPlanName => existingPlanName === newPlanName.trim());
```

Comment:
So we trim whitespace, but should "VmWithAnAwesomeName" be considered the same thing as "vmwithanawesomename"?  Should name duplication gonna be case sensitive?

Reply:
I think the back end does not validate for case sensitivity.  At least this was the case when we were looking at unique mapping names